mocks/mocks.py

```diff
@@ -467,6 +467,15 @@ def mock_image_stack(input_image,
     """
     # measuring the time for stacking images.
     start_time = time.time()
+    try:
+        exptime = exptime.to(u.second)
```

Comment:
I could use `gunagala.ensure_unit` here, but then it would proceed without any error even if the input exposure time was dimensionless.